### PR TITLE
Remove code duplication from Report

### PR DESCRIPTION
**Pull request by salbertson**

### CODE REVIEW

**Comment by gylaz** [comment]
Looks good. What do you think about limiting the amount of weeks to the last 12 or so? That report is getting pretty lengthy. Probably worth of a separate PR though.

File: app/models/report.rb [hunk 1]
```diff
@@ -1,80 +1,72 @@
...
-        generate_output(weekly_activity_sql, week)
-      end
+    generate_report("North Star") do |week|
+      <<-SQL
+        SELECT COUNT(distinct repos.id)
+        FROM repos
+        JOIN builds ON builds.repo_id = repos.id
+        WHERE builds.created_at >= '#{week}'
+        AND builds.created_at < '#{week + 7.days}'
+      SQL
```
Comment (croaky) [inline app/models/report.rb]: Could probably delete this report. It's a dup of https://tbot.io/ns/hound

Comment (gylaz) [inline app/models/report.rb]: How does that spreadsheet pull our data?

Comment (croaky) [inline app/models/report.rb]: magic

Comment (salbertson) [inline app/models/report.rb]: ha, we can remove it in another PR, this is just to remove duplication.

**Comment by salbertson** [comment]
Thanks dudes.

**Pull request merged**